global_route_configurator.js

```
@@ -0,0 +1,79 @@
+'use strict';
+
+const path = require('path');
+const packageJson = require('./package.json');
+
+module.exports = {
+  configureGlobalRoutes: configureGlobalRoutes,
+};
+
+let httpExtension;
+let routeConfiguration;
+
+async function configureGlobalRoutes(container) {
+  httpExtension = await container.resolveAsync('HttpExtension');
+  routeConfiguration = loadConfigJson('http', 'global_routes');
+
+  configureRootRoute();
+  configureAuthorityRoute();
+}
+
+function configureRootRoute() {
+  const packageInfo = getInfosFromPackageJson();
+  httpExtension.app.get('/', (request, response) => {
+    response
+      .status(200)
+      .header('Content-Type', 'application/json')
+      .send(JSON.stringify(packageInfo, null, 2));
+  });
+}
+
+function configureAuthorityRoute() {
+  const iamConfig = loadConfigJson('iam', 'iam_service');
+  const authorityRoute = routeConfiguration.authority ? routeConfiguration.authority : '/authority';
```

please let it crash, if there is no proper config present.

---

otherwise we will wonder, why the the route changes from time to time :)

---

My motivation was, that we offer a set of default routes, which can be modified by the User if he desired it but not enforce the user to supply default routes.

---

@robpal94 To put it simply: I agree with @cmg-dev .